From 612ab0f42c102e8572276ca821efa52c6b8a93b4 Mon Sep 17 00:00:00 2001
From: Mika Laitio <lamikr@gmail.com>
Date: Mon, 28 Oct 2024 01:00:13 -0700
Subject: [PATCH 5/7] fix target already added error

error could happen if tried to
build simulataneously for more than 1 gpu at the same time
(gfx1100, gfx1101)

Signed-off-by: Mika Laitio <lamikr@gmail.com>
---
 example/02_gemm_bilinear/CMakeLists.txt | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/example/02_gemm_bilinear/CMakeLists.txt b/example/02_gemm_bilinear/CMakeLists.txt
index d82c42d5a..97508532c 100644
--- a/example/02_gemm_bilinear/CMakeLists.txt
+++ b/example/02_gemm_bilinear/CMakeLists.txt
@@ -5,9 +5,7 @@ foreach(gpu IN LISTS GPU_TARGETS)
  if(gpu IN_LIST gpu_list1 AND target EQUAL 0)
     add_example_executable(example_gemm_bilinear_wmma_fp16 gemm_bilinear_wmma_fp16.cpp)
     add_example_executable(example_gemm_bilinear_wmma_int8 gemm_bilinear_wmma_int8.cpp)
-endif()
-if(GPU_TARGETS MATCHES "gfx908" OR GPU_TARGETS MATCHES "gfx90a" OR GPU_TARGETS MATCHES "gfx940")
-   set(target 1)
+    set(target 1)
  endif()
 endforeach()
 
-- 
2.43.0

